tests/PHPUnit/Api/ApiTestCase.php

```diff
@@ -33,7 +32,7 @@ protected function setUp(): void
         parent::setUp();
         self::bootKernel();
 
-        $this->manager = $this->getContainer()->get(EntityManagerInterface::class);
+        $this->manager = $this->getContainer()->get('doctrine')->getManager();
```

Comment:
pourquoi ce changement ?

Reply:
Because of 
> The "Doctrine\ORM\EntityManagerInterface" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection instead.

So i read the doc, and i see it on 
https://symfony.com/doc/current/testing/database.html#functional-testing-of-a-doctrine-repository

Reply:
okay, the entityManager is never used in the project so ? Weird...